airflow/kubernetes/pod_generator_deprecated.py

```diff
@@ -178,7 +178,7 @@ def __init__(
 
         self.container.command = cmds or []
         self.container.args = args or []
-        self.container.image_pull_policy = image_pull_policy
+        self.container.image_pull_policy = image_pull_policy if image_pull_policy else "IfNotPresent"
```

Comment:
It's unfortunate it's required now, as this will behave differently... k8s does `IfNotPresent` unless the tag is `latest`, in which case it does `Always`.

Reply:
I actually fixed it to follow this